docs/service-discovery/kubernetes.md

```diff
@@ -17,9 +17,59 @@ First, you need to add the Stork Kubernetes Service Discovery provider:
 </dependency>
 ```
 
-####A few words about server authentication. 
-Stork uses Fabric8 Kubernetes Client to access the Kubernetes resources, concretely the `DefaultKubernetesClient` implementation. It will try to read the ~/.kube/config file in your home directory and load information required for authenticating with the Kubernetes API server. If you are using DefaultKubernetesClient from inside a Pod, it will load ~/.kube/config from the ServiceAccount volume mounted inside the Pod. You can override this configuration if you want a more complex configuration.
+#### A few words about server authentication. 
+Stork uses [Fabric8 Kubernetes Client](https://github.com/fabric8io/kubernetes-client#readme) to access the Kubernetes resources, concretely the `DefaultKubernetesClient` implementation. 
 
+It will try to read the `~/.kube/config` file from your local machine and load the token for authenticating with the Kubernetes API server.
+
+If you are using the Stork Kubernetes discovery provider from inside a _Pod_, it loads `~/.kube/config` from the container file system.
+
```

Comment:
Wondering about the rendering. Would it create new single-line paragraphs? 

In general, I use the 1 sentence per-line rule (ease edits and reviews), but no blank lines between lines from the same paragraph.

Reply:
maybe we could integrate generating documentation previews into PR jobs...

Reply:
I don't get to avoid the blank line between lines. If I don't let the blank line, the paragraph is not split into lines....